spirv_cross_c.cpp

```diff
@@ -1937,6 +1937,18 @@ SpvExecutionModel spvc_compiler_get_execution_model(spvc_compiler compiler)
 	return static_cast<SpvExecutionModel>(compiler->compiler->get_execution_model());
 }
 
+SPVC_PUBLIC_API void spvc_compiler_update_active_builtins(spvc_compiler compiler)
```

Comment:
Don't need SPVC_PUBLIC_API here, header takes care of it.

Reply:
Done